packages/perseus-core/src/utils/grapher-util.ts

```diff
@@ -0,0 +1,644 @@
+import _ from "underscore";
+
+import type {
+    LinearType,
+    QuadraticType,
+    SinusoidType,
+    TangentType,
+    ExponentialType,
+    LogarithmType,
+    AbsoluteValueType,
+    Coords,
+} from "./grapher-types";
+import type {Coord} from "../data-schema";
+
+export const MOVABLES = {
+    PLOT: "PLOT",
+    PARABOLA: "PARABOLA",
+    SINUSOID: "SINUSOID",
+};
```

Comment:
When I moved this, I couldn't move `Graphie` which was used by these objects to map `Graphie` components to their `Movable` properties. Instead I made this enough, changed `Movable: Graphie.Plot` to `movable: MOVABLES.PLOT`, and mapped them back to Graphie widgets using `movableToGraphieMovable`.